Wizardry/Core/BattleSys/Source/BattleCalcReal.c

```diff
@@ -166,6 +166,14 @@ void ComputeBattleUnitEffectiveStats(struct BattleUnit * attacker, struct Battle
     ComputeBattleUnitSilencerRate(attacker, defender);
     ComputeBattleUnitSpecialWeaponStats(attacker, defender);
 
+#if (defined(SID_NoGuard) && (SID_NoGuard < MAX_SKILL_NUM))
+        if (SkillTester(&attacker->unit, SID_NoGuard))
+        {
+            attacker->battleEffectiveHitRate = 100;
+            attacker->battleAvoidRate = -100;
```

Comment:
This will cause this value to directly become negative. There should be no subsequent corrections to modify the negative number to 0. This should cause ui problems.

Reply:
It's the very last thing that happens in `BattleCalcReal.c` so there shouldn't be anything else modifying stats? 

Besides, I've tested it and even when the avoid is negative it doesn't cause any UI problems for the avoid value in battle or when displaying the stat screen. It's always positive up to the point the forecast menu is displayed.